lib/twine/formatters/android.rb

```diff
@@ -217,7 +217,6 @@ def format_section(section, lang)
           else
             # If the plural section has no definitions in this language, skip it
             # Otherwise, only output the correct definitions of the correct language
-            definitions.delete_if { |definition| definition.translation_for_lang_or_nil(lang, @twine_file.language_codes[0]).nil? }
```

Comment:
@imbryk can you test it when the translation in the text file is empty. Does it skip adding it the the localized output file?

Reply:
@aminelaadhari we actually manually override this behaviour in our generate_android.sh scripts, by setting `dev-language` to the same as the translation language:
https://github.com/thefabulous/fabulous-core/blob/master/strings/generate_android.sh#L23
`twine generate-localization-file strings_android.txt res/values-fr/strings_android.xml --lang fr --developer-language fr --format android`

Reply:
@imbryk that's the main issue: --developer-language  should be always EN no?

Reply:
ios don't use --developer-language

https://github.com/thefabulous/fabulous-core/blob/master/strings/generate_ios.sh

Reply:
@aminelaadhari that's not the main issue - that's a different issue (if it is one) ;) 
i.e. this does not result in current error, just removing the `--developer-language ` keeps the same error

Reply:
so to confirm - If I remove the `--developer-language` a lot of duplicates are removed, i.e. we have a lot of english words in our translation files. I assume this is to stop lint from complaining, and pretend that we do have the translations

this still works fine for non-plurals after the removal of this single line
this line only affects plurals

Reply:
https://github.com/thefabulous/fabulous-core/commit/15b6ff001f9d092b82dd43c622da1e3683e9aebc

This is when the switch has happened.

IMO To fix the linter, we should duplicate the translations in the source files and not do a workaround like this.

@imbryk @kmalmur I will let you decide on this as you already have the knowledge now :)